src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -177,10 +189,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $retryStrategies[$receiverName] = null !== $this->retryStrategyLocator ? $this->retryStrategyLocator->get($receiverName) : null;
         }
 
-        if (null !== $input->getOption('bus')) {
-            $bus = $this->busLocator->get($input->getOption('bus'));
-        } else {
-            $bus = new RoutableMessageBus($this->busLocator);
+        if (null !== $busName = $input->getOption('bus')) {
+            $bus = $this->routableBus ? $this->routableBus->getMessageBus(new Envelope(new class {}, [new BusNameStamp($busName)])) : $this->busLocator->get($busName);
```

Comment:
woh. I had been picturing something like a new `getMessageBus(string $busName)`... since we have to add a new method either way, maybe a string argument is cleaner?

Reply:
done :)